interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -1329,6 +1329,10 @@ namespace {
       argvCompile.push_back("-fno-omit-frame-pointer");
 #endif
 
+    // Disable optimizations and keep frame pointer when debugging
+    if (std::getenv("CLING_DEBUG"))
```

Comment:
If I ran with `CLING_DEBUG=0`, this would still go to `-O0`. I would find the surprising as a user.

Reply:
Yes, I documented that setting it to anything enabled it, but I guess we should try to interpret common true/false things like yes/no, true/false, 0/1, on/off. Are there other things in ROOT that do that that I could reuse?